src/styles/tailwind.css

```diff
@@ -22,32 +21,49 @@
   --media-background-color: --_primary-color;
 }
 
-.prose :where(img):not(:where([class~="not-prose"],[class~="not-prose"] *)) {
-  @apply mt-0 mb-0
+.prose :where(img):not(:where([class~='not-prose'], [class~='not-prose'] *)) {
+  @apply mb-0 mt-0;
 }
 
-.prose :where(code):not(:where([class~="not-prose"],[class~="not-prose"] *))::after, .prose :where(code):not(:where([class~="not-prose"],[class~="not-prose"] *))::before {
-  content: "";
+.prose
+  :where(code):not(:where([class~='not-prose'], [class~='not-prose'] *))::after,
+.prose
+  :where(code):not(
+    :where([class~='not-prose'], [class~='not-prose'] *)
+  )::before {
+  content: '';
 }
 
-.prose p > code, .prose li > code, .prose h2 > code {
-  @apply border border-opacity-30 border-gray-500 rounded-md px-1 py-1 mx-0.5 bg-gray-500 bg-opacity-10 font-semibold text-nowrap
+.prose p > code,
+.prose li > code,
+.prose h2 > code {
+  @apply mx-0.5 text-nowrap rounded-md border border-gray-500 border-opacity-30 bg-gray-500 bg-opacity-10 px-1 py-1 font-semibold;
 }
 
 .prose h2 {
-  margin-top: 4rem ;
+  margin-top: 4rem;
+}
+
+.prose h3 {
```

Comment:
This is the only bit I changed

Reply:
https://github.com/replayio/docs/pull/199